cli/src/etos_client/sse/protocol.py

```diff
@@ -98,6 +98,34 @@ def __str__(self):
         return self.message.get("message")
 
 
+class Report(UserEvent):
+    """An ETOS test case report file event."""
+
+    def __init__(self, event: dict) -> None:
+        """Initialize a report by loading an expected json string."""
+        super().__init__(event)
+
+        try:
+            self.file = json.loads(self.data)
```

Comment:
Add checksum? To minimize the risk of files becoming corrupt during upload/download?

Reply:
Added checksum